traitsui/testing/tester/ui_tester.py

```diff
@@ -131,7 +132,7 @@ class App(HasTraits):
         of decreasing priority. A shallow copy will be made.
     """
 
-    def __init__(self, registries=None):
+    def __init__(self, registries=None, delay=0):
```

Comment:
Oops, I think I failed to push a comment :(
Could we expand the description in the class docstring what this `delay` is about please?
In particular, what unit did you choose?

I realized that the "Attributes" section should really be named "Parameters" and there needs to be a separate "Attributes" section to conform to NumPy docstring convention. The `UIWrapper`'s `Parameters` needs to be moved up into the class docstring too. I can do the moves in a separate PR though. If you could just add the description for the `delay` next to `registries`, that'd be great.